src/interfaces/keyIterator.ts

```diff
@@ -25,6 +25,8 @@ export interface KeyIterator {
     label: string
     privateKey: string
     dedicatedToOneSA: boolean
-    meta: null
+    meta: {
+      timestamp: number
+    }
```

Comment:
Also we should discuss a migration that will change meta: null to an object for the current data?